tox.ini

```diff
@@ -16,7 +16,7 @@ all_path = {[vars]src_path} {[vars]unit_test_path} {[vars]integration_test_path}
 
 [testenv]
 set_env =
-    PYTHONPATH = {toxinidir}:{toxinidir}/lib:{[vars]src_path}
+    PYTHONPATH = {toxinidir}:{toxinidir}/lib:{[vars]src_path}:{[vars]integration_test_path}/vault_kv_requirer_operator/src
```

Comment:
This gets rid of the need for a `# type: ignore` in the test code that masked one of my errors -- alternative would be to break up the `pyright` calls and only provide this path when type checking the specific file.